Common/src/main/java/com/blamejared/crafttweaker/api/logging/CommonLoggers.java

```diff
@@ -0,0 +1,90 @@
+package com.blamejared.crafttweaker.api.logging;
+
+import com.blamejared.crafttweaker.api.CraftTweakerAPI;
+import com.blamejared.crafttweaker.api.CraftTweakerConstants;
+import com.google.common.base.Suppliers;
+import org.apache.logging.log4j.Logger;
+
+import java.util.function.Supplier;
+
+/**
+ * Provide a series of {@link Logger} instances for the most common systems.
+ *
+ * <p>Usage of these loggers is encouraged whenever possible to ensure consistency in the logging. Refer to the
+ * documentation of the various methods for additional details relating to the specific system.</p>
+ *
+ * @see CraftTweakerAPI#getLogger(String)
+ * @since 10.1.0
+ */
+public final class CommonLoggers {
+    
+    private static final Supplier<Logger> COMMANDS = loggerSupplier(ctSystem("Commands"));
+    private static final Supplier<Logger> OWN = loggerSupplier(CraftTweakerConstants.MOD_NAME);
```

Comment:
Ideally this would have a better name, sure it is "ours" but it may be confusing for others.